flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/NettyShuffleEnvironmentConfiguration.java

```diff
@@ -480,6 +493,7 @@ public int hashCode() {
         result = 31 * result + networkBufferSize;
         result = 31 * result + partitionRequestInitialBackoff;
         result = 31 * result + partitionRequestMaxBackoff;
+        result = 31 * result + partitionRequestNotifyTimeout.hashCode();
```

Comment:
you forgot to modify `equals` method as well.

Reply:
Done